app/Providers/AppServiceProvider.php

```diff
@@ -14,7 +17,16 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        if ($this->app->environment('local')) {
```

Comment:
I would suggests to keep still if in the bind. From other point of view I requested just check for production, so all other environments should receive the DebugRequestActivityLog